task/buildah-oci-ta/0.2/buildah-oci-ta.yaml

```
@@ -454,13 +458,25 @@ spec:
         # Prevent ShellCheck from giving a warning because 'image' is defined and 'IMAGE' is not.
         declare IMAGE
 
-        unshare -Uf $UNSHARE_ARGS --keep-caps -r --map-users 1,1,65536 --map-groups 1,1,65536 -w ${SOURCE_CODE_DIR}/$CONTEXT -- buildah build \
-          $VOLUME_MOUNTS \
-          "${BUILDAH_ARGS[@]}" \
-          "${LABELS[@]}" \
-          --tls-verify=$TLSVERIFY --no-cache \
-          --ulimit nofile=4096:4096 \
-          -f "$dockerfile_copy" -t "$IMAGE" .
+        buildah_cmd_array=(
+            buildah build
+            "${VOLUME_MOUNTS[@]}"
+            "${BUILDAH_ARGS[@]}"
+            "${LABELS[@]}"
+            --tls-verify="$TLSVERIFY" --no-cache
+            --ulimit nofile=4096:4096
+            -f "$dockerfile_path" -t "$IMAGE" .
```

```suggestion
            -f "$dockerfile_copy" -t "$IMAGE" .
```
here and in the rest of the tasks.

---

This has been resolved with the latest commits.